From avorontsov@ru.mvista.com  Tue Dec 22 11:52:20 2009
From: Anton Vorontsov <avorontsov@ru.mvista.com>
Date: Mon, 14 Dec 2009 18:41:05 +0300
Subject: USB: ehci-fsl: Fix sparse warnings
To: Greg Kroah-Hartman <gregkh@suse.de>
Cc: Scott Wood <scottwood@freescale.com>, Dave Liu <daveliu@freescale.com>, Jerry Huang <Chang-Ming.Huang@freescale.com>, linux-usb@vger.kernel.org, linuxppc-dev@ozlabs.org
Message-ID: <20091214154105.GA26805@oksana.dev.rtsoft.ru>
Content-Disposition: inline


This patch fixes following warnings:

ehci-fsl.c:43:5: warning: symbol 'usb_hcd_fsl_probe' was not declared. Should it be static?
ehci-fsl.c:150:6: warning: symbol 'usb_hcd_fsl_remove' was not declared. Should it be static?

Signed-off-by: Anton Vorontsov <avorontsov@ru.mvista.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ehci-fsl.c |    7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

--- a/drivers/usb/host/ehci-fsl.c
+++ b/drivers/usb/host/ehci-fsl.c
@@ -40,8 +40,8 @@
  * Allocates basic resources for this USB host controller.
  *
  */
-int usb_hcd_fsl_probe(const struct hc_driver *driver,
-		      struct platform_device *pdev)
+static int usb_hcd_fsl_probe(const struct hc_driver *driver,
+			     struct platform_device *pdev)
 {
 	struct fsl_usb2_platform_data *pdata;
 	struct usb_hcd *hcd;
@@ -147,7 +147,8 @@ int usb_hcd_fsl_probe(const struct hc_dr
  * Reverses the effect of usb_hcd_fsl_probe().
  *
  */
-void usb_hcd_fsl_remove(struct usb_hcd *hcd, struct platform_device *pdev)
+static void usb_hcd_fsl_remove(struct usb_hcd *hcd,
+			       struct platform_device *pdev)
 {
 	usb_remove_hcd(hcd);
 	iounmap(hcd->regs);
